tidoop-mr-lib-api/src/server.js

```diff
@@ -23,36 +23,134 @@
  * Author: frb
  */
 
-// requires
+// module dependencies
 var Hapi = require('hapi');
-var config = require('../conf/tidoop-mr-lib-api.json');
-var pjson = require('../package.json');
+var run = require('./cmd_runner.js').run;
+var version = require('../package.json').version;
+var port = require('../conf/tidoop-mr-lib-api.json').port;
+var tidoopMRLibPath = require('../conf/tidoop-mr-lib-api.json').tidoopMRLibPath;
+var tidoopMysql = require('./mysql_driver.js');
+var serverUtils = require('./server_utils.js');
 
 // create a server with a host and port
 var server = new Hapi.Server();
 
 server.connection({ 
     host: 'localhost',
-    port: config.port
+    port: port
 });
 
-// add the route
+// create a connection to the MySQL server
+tidoopMysql.connect();
+
+// add routes
 server.route({
     method: 'GET',
-    path:'/version',
+    path: '/tidoop/v1/version',
     handler: function (request, reply) {
-        console.log("Request: GET /version");
-        var response = '{version: ' + pjson.version + '}';
+        console.log("Request: GET /tidoop/v1/version");
+        var response = '{version: ' + version + '}';
         console.log("Response: " + response);
         reply(response);
-    }
+    } // handler
+});
+
+server.route({
+    method: 'GET',
+    path: '/tidoop/v1/jobs/',
+    handler: function(request, reply) {
```

Comment:
I don't really get this route :) Do you plan to implemented in the future? On the other hand, I guess you have tested it from the client side and throwing an Error is notified to the calling client as some kind of 5XX code??? :) When notifying errors to the client side using hapi I typically compose the corresponding `boom` error and reply it to the client side :) You can see an example of this here: http://hapijs.com/api#error-response